modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/metadata/IgniteMdFragmentMapping.java

```diff
@@ -216,7 +217,7 @@ public FragmentMapping fragmentMapping(IgniteTableFunctionScan rel, RelMetadataQ
      * @param mq  Metadata query instance.
      * @return Fragment meta information.
      */
-    public static FragmentMapping _fragmentMapping(RelNode rel, RelMetadataQuery mq) {
+    public static FragmentMapping fragmentMappingForMetadataQuery(RelNode rel, RelMetadataQuery mq) {
```

Comment:
@korlov42 could you please check the rename.

Reply:
It would be better to rename to `mapFragment`